### DISCUSSION TITLE
Facing issue with the compilation of LISF_557WW_release_7.4.2 LIS

### DISCUSSION THREAD

**Discussion by ranjit02** [NASA-LIS/LISF#1283] [General]
Hii All,
I am compiling the LISF_557WW_release_7.4.2 LIS part, and I am facing an issue with the error message given below:
``

Compiling LIS

[INFO] Compiling LIS source code

ftn -c  -O2 -fp-model precise -u -traceback -fpe0 -nomixed_str_len_arg -names lowercase -convert big_endian -assume byterecl -el/ESMF/esmf-ESMF_7_1_0r/mod -DUSE_INCLUDE_MPI -I/home/apps/SiteSoftwares/intel/ECCODES/2.21.0/include/ -I/opt/cray/pe/netcdf/I/home/apps/SiteSoftwares/intel/NCEPLIBS/crtm-2.3.0/lib/include/ -I/include/ -DLIS_JULES -I./ -I../core/ -I../params/gfrac/SPOcs/ ../surfacemodels/land/jules.5.0/utils/mpi_dummy/mpi_mod.F90
../surfacemodels/land/jules.5.0/utils/mpi_dummy/mpi_mod.F90(438): error #7111: An interface-body is incorrectly positioned wit
interface mpi_bcast
----------^
compilation aborted for ../surfacemodels/land/jules.5.0/utils/mpi_dummy/mpi_mod.F90 (code 1)
Makefile:139: recipe for target 'mpi_mod.o' failed
make: *** [mpi_mod.o] Error 1

[ERR] Compile failed
--------------------------------------------------------------------------------------------------------------------------
could any one knows about it please guide me on this, any help will be appreciated.

**Comment by dmocko** [comment]
Hello @ranjit02 

The 557WW releases are not for general use.  You need separate permission to use the JULES LSM, including how to obtain access to the model code.  I recommend that you use the latest LISF public release, which is version 7.4.1.

**Reply by ranjit02** [reply]
Thanks for your response @dmocko
